client/src/store/modules/i18n/en.js

```diff
@@ -28,5 +28,18 @@ export default {
     NEW_PASSWORD: 'New password',
 
     YOU_LOG_IN_WITH_AN_EMAIL_ADDRESS_AND_PASSWORD:
-        'You log in with an email address and password'
+        'You log in with an email address and password',
+
+    CREATE_AN_ACCOUNT: 'Create an account',
+    ALREADY_REGISTERED: 'Already registered?',
+    LOG_IN: 'Log in',
+    FULL_NAME: 'Full Name',
+    NAME: 'Name',
+    CONFIRM_PASSWORD: 'Confirm Password',
+    SIGN_UP: 'Sign Up',
+    SUCCESSFULLY_REGISTERED: 'Successfully registered! Now you can authorize!',
+    PASSWORDS_DONT_MATCH: "Passwords don't match",
+    fieldIsRequired(field = '') {
```

Comment:
it is a bad idea, here must be only constants, no functions. Create constant FIELD_REQUIRED: '${field} is required', then do replace lang.FIELD_REQUIRED.replace(/\$\{field\}/, field). And as I see, you do not use it anyway

Reply:
Why, function is bad idea?

Reply:
because we are going to move them to vuex state. And also, you mentioned that it is better to create .json, and it was a good proposal. The main reason is that localization is filled not always up by techy people, so it would be difficult for them to edit functions. And furthermore, sometimes, it can be done with some tools. So it is a good practice to keep localization as just static constants. After all, the KISS principle fits here (https://en.wikipedia.org/wiki/KISS_principle)